ports/servoshell/minibrowser.rs

```diff
@@ -81,17 +84,29 @@ impl Minibrowser {
             TopBottomPanel::top("toolbar").show(ctx, |ui| {
                 ui.allocate_ui_with_layout(
                     ui.available_size(),
-                    egui::Layout::right_to_left(egui::Align::Center),
+                    egui::Layout::left_to_right(egui::Align::Center),
                     |ui| {
-                        if ui.button("go").clicked() {
-                            event_queue.borrow_mut().push(MinibrowserEvent::Go);
-                            location_dirty.set(false);
+                        if ui.button("back").clicked() {
+                            event_queue.borrow_mut().push(MinibrowserEvent::Back);
+                        }
+                        if ui.button("forward").clicked() {
+                            event_queue.borrow_mut().push(MinibrowserEvent::Forward);
                         }
 
+                        // As we are using the `left_to_right` layout, we reserve space for the "Go" button
+                        // at the end by subtracting its estimated width from the available width.
+                        let button_space = 30.0;
+                        let text_edit_width = ui.available_width() - button_space;
```

Comment:
we can avoid hardcoding the width by wrapping the location bar and go button in a nested allocate_ui_with_layout using a right-to-left layout :)

Reply:
thanks for the suggestion. Updated in [3162fd3](https://github.com/servo/servo/pull/30805/commits/3162fd3438600a6502b6340f40490383915a1c17)